spec/schema.json

```diff
@@ -285,6 +285,45 @@
         }
       }
     },
+    "/transactions/{transaction_id}/confirmation-statement/{confirmation_statement_id}/registered-email-address": {
```

Comment:
Should this path be removed? think the private get email address replaces this now

Reply:
done